ci/integration/scripts/create-app.py

```diff
@@ -15,12 +15,12 @@ def createAPP(appName):
 def getAppClient(appName):
     url = SQUIDEX_URL + "/api/apps/" + appName + "/clients"
     r = requests.get(url = url, headers = headers)
-    json_data = r.json() 
+    json_data = r.json()
     return json_data['items'][0]
 
 def promoteClient(appName, client):
     # Make user owner to allow him deleting the app
-    clientUrl = SQUIDEX_URL + "/api/apps/" + appName + "/clients/" + client['id'] 
+    clientUrl = SQUIDEX_URL + "/api/apps/" + appName + "/clients/" + client['id']
```

Comment:
You could use the f-strings (if this is python 3.6 and above).
https://realpython.com/python-f-strings/